Week3/homework/4-step3.js

```diff
@@ -1,2 +1,19 @@
 'use strict';
-// paste your freeCodeCamp solutions in here
+//----------1
+{
+  function Dog() {
+    this.name = 'Jäger';
+    this.color = 'black and white';
+    this.numberOfLegs = 4;
+  }
+
+  const Hound = new Dog();
```

Comment:
Variable, function and parameter names should be in camelCase. Only constructor functions (function that you call in combination with the keyword `new`) and ES6 class names (not yet covered in the lectures) should be in PascalCase.

So this would be correct:

```js
const hound = new Dog();
```

`Dog()` is a constructor function. `hound` is an ordinary variable name.